app/views/trainees/index.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= render PageTitle::View.new(title: trainees_page_title(@paginated_trainees)) %>
```

Comment:
As per my comment above, this will become:

```
<%= render PageTitle::View.new(title: trainees_page_title(@paginated_trainees, @total_trainees_count)) %>
```

once you've defined the new instance variable.